src/socket/unix_server.cr

```diff
@@ -43,8 +45,8 @@ class UNIXServer < UNIXSocket
   # server socket when the block returns.
   #
   # Returns the value of the block.
-  def self.open(path, type : Type = Type::STREAM, backlog = 128)
-    server = new(path, type, backlog)
+  def self.open(path, type : Type = Type::STREAM, backlog = 128, abstract is_abstract = false)
```

Comment:
I'd like, but I can't: 
when I use `abstract` in:
```
server = new(path, type, backlog, abstract)
```
I get the error saying that I cannot use `abstract` inside a `def`